examples/resnet/resnet_main.py

```diff
@@ -16,8 +16,10 @@
 import resnet_model
 
 # Tensorflow must be at least version 1.2.0 for the example to work.
-if (int(tf.__version__.split(".")[0]) < 1) and (int(tf.__version__.split(".")[1]) < 2):
-    raise Exception("Your Tensorflow version is less than 1.0.0. Please "
+tf_major = int(tf.__version__.split(".")[0])
+tf_minor = int(tf.__version__.split(".")[1])
+if (tf_major < 1) or (tf_major == 1 and tf_minor < 2):
+    raise Exception("Your Tensorflow version is less than 1.2.0. Please "
```

Comment:
@Wapaul1 I changed the logic a bit here (in particular `and` => `or` and also some other stuff). Let me know if this is correct.